pkgs/applications/graphics/apitrace/default.nix

```diff
@@ -1,27 +1,37 @@
-{ lib, stdenv, fetchFromGitHub, cmake, libX11, procps, python2, libdwarf, qtbase, qtwebkit, wrapQtAppsHook, libglvnd }:
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  pkg-config,
+  libX11,
+  procps,
+  python3,
+  libdwarf,
+  qtbase,
+  wrapQtAppsHook,
+  libglvnd,
+  gtest,
+  brotli,
+}:
 
 stdenv.mkDerivation rec {
   pname = "apitrace";
-  version = "7.1-572-g${builtins.substring 0 8 src.rev}";
+  version = "11.1";
 
   src = fetchFromGitHub {
-    sha256 = "11bwb0l8cr1bf9bj1s6cbmi77d5fy4qrphj9cgmcd8jpa862anp5";
-    rev = "26966134f15d28f6b4a9a0a560017b3ba36d60bf";
-    repo = "apitrace";
     owner = "apitrace";
+    repo = "apitrace";
+    rev = version;
+    hash = "sha256-rvC6iVWNNxH11hzQvRTo+SQi9jEUCPWGSdJmKJe9SQ0=";
+    fetchSubmodules = true;
```

Comment:
We should at least use our zlib, libpng and brotli.

Reply:
I believe it's using those to link statically so we can't really avoid that (unless we do some bad pkgsStatic things)

Reply:
Maybe that would be better than trusting upstream to update those dependencies and fix CVEs?